assets/queries/terraform/azure/app_service_authentication_disabled/query.rego

```diff
@@ -0,0 +1,33 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
```

Comment:
format rego query either using `opa fmt` [link](https://github.com/open-policy-agent/opa/blob/c6344d05256c2ceb6ad4db3feb88922d44649862/cmd/fmt.go) or by using vscode OPA extension and enabling format on save on settings